components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -107,6 +116,8 @@ const val SCOPE_PROFILE = "profile"
 const val SCOPE_SYNC = "https://identity.mozilla.com/apps/oldsync"
 // Necessary to obtain a sessionToken, which gives full access to the account.
 const val SCOPE_SESSION = "https://identity.mozilla.com/tokens/session"
+// Not allowed to check authorization status more frequently than 10 seconds.
+const val AUTH_CHECK_CIRCUIT_BREAKER_WINDOW_MS = 1000L * 10
```

Comment:
As you mentioned in slack, it's hard to know what the right value is here. I worry that 10 seconds may be too short, depending on how far removed the failing action is from the recovery process here. How would you feel about 30 seconds or even 1 minute? Our OAuth tokens should all live *much* longer than that in practice!

The tradeoff I guess, is that we increase the risk of spuriously driving the user to the "unauthenticated" state if they happen to trigger several legitimate auth errors within this time period.